website/content/docs/install-boundary/configure-workers.mdx

```diff
@@ -305,6 +318,122 @@ If present, `boundary server` starts a worker subprocess.
 
 Refer to the documentation for additional [top-level configuration options](/boundary/docs/configuration) and additional [controller-specific options](/boundary/docs/configuration/controller).
 
+</Tab>
+<Tab heading="Community Edition">
+
+For the Community Edition of Boundary, you must configure a worker server that communicates with the controller and is responsible for storage-related tasks.
+
+Complete the following steps to configure the worker.
```

Comment:
Not sure about the exact phrasing here, but it would be nice to include this additional note

```suggestion
For the Community Edition of Boundary, you must configure a worker server that communicates with the controller and is responsible for storage-related tasks.

Boundary Community Edition only supports egress workers. To configure workers for multi-hop sessions, check the Boundary Enterprise configuration.

Complete the following steps to configure the worker.
```